migtests/scripts/mysql/env.sh

```diff
@@ -1,4 +1,4 @@
-export SOURCE_DB_HOST=${SOURCE_DB_HOST:-"127.0.0.1"}
+export SOURCE_DB_HOST=${SOURCE_DB_HOST:-"localhost"}
 export SOURCE_DB_PORT=${SOURCE_DB_PORT:-3306}
-export SOURCE_DB_USER=${SOURCE_DB_USER:-"root"}
-export SOURCE_DB_PASSWORD=${SOURCE_DB_PASSWORD:-"root"}
+export SOURCE_DB_USER=${SOURCE_DB_USER:-"ybvoyager"}
```

Comment:
For all source databases, let's define two sets of env vars for two users:
- `SOURCE_DB_ADMIN_USER` and `SOURCE_DB_ADMIN_PASSWORD`
- `SOURCE_DB_USER` and `SOURCE_DB_PASSWORD`

The admin user credentials are used for activities like creating/dropping databases, granting permissions, etc.

The later set is used for migration. Default value of the `SOURCE_DB_USER` should be `ybvoyager`.

Reply:
Done